pkg/docker/docker.go

```diff
@@ -1143,8 +1110,7 @@ func (d *stiDocker) BuildImage(opts BuildImageOptions) error {
 		dockerOpts.CPUQuota = opts.CGroupLimits.CPUQuota
 	}
 	glog.V(2).Infof("Building container using config: %+v", dockerOpts)
-	ctx, cancel := getDefaultContext(((1<<63 - 1) * time.Nanosecond)) // infinite duration ... go does not expost max duration constant
-	defer cancel()
+	ctx := context.Background() // no timeout
```

Comment:
oops .. meant to delete this comment.  Or maybe I did in another spot.  In any even, the run container paths is special case - leave it infinite.  Apologies for the churn.